lib/screens/v2/candidate_generator/dup.ex

```diff
@@ -0,0 +1,78 @@
+defmodule Screens.V2.CandidateGenerator.Dup do
+  @moduledoc false
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.Dup
+  alias Screens.Config.V2.Header.CurrentStopId
+  alias Screens.Stops.Stop
+  alias Screens.V2.CandidateGenerator
+  alias Screens.V2.Template.Builder
+  alias Screens.V2.WidgetInstance.{NormalHeader, Placeholder}
+
+  @behaviour CandidateGenerator
+
+  @impl CandidateGenerator
+  def screen_template do
+    {:screen,
+     %{
+       screen_normal: [
+         {:rotation_zero,
```

Comment:
The main reason I used words instead of digits is it felt cleaner to me in most situations. I think the digit is fine in an atom, but in filenames and frontend variables it felt less so for some reason. I don't feel super strongly about that though so feel free to push back.